week2/코딩도장-13.7-심사문제.py

```diff
@@ -0,0 +1,20 @@
+# 13.7 심사문제: 온라인 할인 쿠폰 시스템 만들기
+
+# 표준 입력으로 가격(정수)과 쿠폰 이름이 각 줄에 입력됩니다. 
+# Cash3000 쿠폰은 3,000원, Cash5000 쿠폰은 5,000원을 할인합니다. 
+# 쿠폰에 따라 할인된 가격을 출력하는 프로그램을 만드세요
+# (input에서 안내 문자열은 출력하지 않아야 합니다).
+
+# 입력
+# 27000
+# Cash3000
+# 결과
+# 24000
+
+x = int(input())
+y = input()
```

Comment:
변수명은 의미를 파악할 수 있게 지어야 합니다!
따라서 x와 y 대신 pay와 coupon 등으로 바꾸는 것이 좋아 보입니다.
```suggestion
pay = int(input())
coupon = input()
```